oidb/pepper/service/src/lib.rs

```diff
@@ -23,24 +23,20 @@ pub type KeyID = String;
 
 /// The core processing logic of this pepper service.
 pub async fn process(request: PepperRequest) -> PepperResponse {
-    match request {
-        PepperRequest::V0(req) => {
```

Comment:
I think having the API version in request body is slightly better than in request url: one single bcs::from_bytes take care of both version validation and request body validation.

@heliuchuan wdyt? In frontend it should be no difference?

Reply:
I just think it's ugly lol.  I think when we add encryption just add it as part of the response and then the SDK will switch over, and then you can remove it later and people will have to upgrade.

Ideally we don't have any breaking change API version updates haha (perhaps wishful thinking)